From 3c3652ee672179f16546dae3cd473231f44d0271 Mon Sep 17 00:00:00 2001
From: ahs <amrita.h.s@intel.com>
Date: Thu, 13 Dec 2018 14:15:05 +0530
Subject: [PATCH] Have GenCopy as the default GC.

Ported original gerrit link:
https://android.intel.com/#/c/621338/

Change-Id: Id24d509a6ea8f3e3a2f8d100ff380915412862b2
Category: device enablement
Domain: AOSP.ART-Other
Origin: internal
Upstream-Candidate: no, proprietary
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-73559
Signed-off-by: ahs <amrita.h.s@intel.com>
---
 core/soong_config.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/soong_config.mk b/core/soong_config.mk
index f8cb2fb..153ac43 100644
--- a/core/soong_config.mk
+++ b/core/soong_config.mk
@@ -123,6 +123,7 @@ $(call add_json_list, DeviceSystemSdkVersions,           $(BOARD_SYSTEMSDK_VERSI
 $(call add_json_list, Platform_systemsdk_versions,       $(PLATFORM_SYSTEMSDK_VERSIONS))
 $(call add_json_bool, Malloc_not_svelte,                 $(call invert_bool,$(filter true,$(MALLOC_SVELTE))))
 $(call add_json_str,  Override_rs_driver,                $(OVERRIDE_RS_DRIVER))
+$(call add_json_str,  ArtDefaultGCType,                  $(if $(filter "","$(ART_DEFAULT_GC_TYPE)"),CC,$(ART_DEFAULT_GC_TYPE)))
 
 $(call add_json_bool, Treble_linker_namespaces,          $(filter true,$(PRODUCT_TREBLE_LINKER_NAMESPACES)))
 $(call add_json_bool, Enforce_vintf_manifest,            $(filter true,$(PRODUCT_ENFORCE_VINTF_MANIFEST)))
-- 
1.9.1

